src/main/java/org/kohsuke/github/internal/EnumUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.kohsuke.github.internal;
+
+import java.util.Locale;
+
+public final class EnumUtils {
+
+    public static <E extends Enum<E>> E getEnum(Class<E> enumClass, String value, E defaultEnum) {
+        if (value == null) {
+            return null;
+        }
```

Comment:
Why return `null`?  Enumerations are value types, I would expect this method to always return a value, never null. 

A comment on you reasoning here would be useful.

Reply:
Maybe change the method name to `getEnumOrNull`?  ... I'd kind of expect the name to be `getEnumOrDefault` already, so then it would become `getNullableEnumOrDefault`?   It's "internal" so we can change it later if needed, I guess.

Reply:
null is a valid value for some of the fields that are enums. When the API doesn't return a value, we want it to be null.

I adjusted the method name per your suggestion.